cmd/generate/config/rules/octopusdeploy.go

```diff
@@ -0,0 +1,32 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func OctopusDeployApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "octopus-deploy-api-key",
+		Description: "Discovered a potential Octopus Deploy API key, risking application deployments and operational security.",
+		Regex:       utils.GenerateUniqueTokenRegex(`API-[A-Z0-9]{26}`, false),
```

Comment:
@rgmz I'm concerned this would lead to some false positives since `API-` could be fairly common. We can keep an eye on this rule. Probably fine as is though.